dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+biddercode: exadsadserver
+gvl_id: 1084
+tcfeu_supported: true
+media_types: banner, video, native
+gpp_sids: tcfeu
```

Comment:
I couldn't find any trace of gpp in your adapter  code. Only gdpr is handled here
https://github.com/prebid/Prebid.js/pull/11284/files#diff-7b07170acdd41519116c203893aece137dd33e19f8475b5c7d806479b139de6fR68-R72

Reply:
I confirm that for now we'll support only gdpr. Removed gpp_sids param.